bundles/create_openhab_binding_skeleton.cmd

```diff
@@ -19,18 +20,20 @@ SET GithubUser=%~3
 
 call :LoCase BindingIdInLowerCase
 
-call mvn -s archetype-settings.xml archetype:generate -N -DarchetypeGroupId=org.openhab.core.tools.archetypes -DarchetypeArtifactId=org.openhab.core.tools.archetypes.binding -DarchetypeVersion=%OpenhabVersion% -DgroupId=org.openhab.binding -DartifactId=org.openhab.binding.%BindingIdInLowerCase% -Dpackage=org.openhab.binding.%BindingIdInLowerCase% -Dversion=%OpenhabVersion% -DbindingId=%BindingIdInLowerCase% -DbindingIdCamelCase=%BindingIdInCamelCase% -DvendorName=openHAB -Dnamespace=org.openhab -Dauthor="%Author%" -DgithubUser="%GithubUser%"
+call mvn -s archetype-settings.xml archetype:generate -N -DarchetypeGroupId=org.openhab.core.tools.archetypes -DarchetypeArtifactId=org.openhab.core.tools.archetypes.binding -DarchetypeVersion=%OpenhabCoreVersion% -DgroupId=org.openhab.binding -DartifactId=org.openhab.binding.%BindingIdInLowerCase% -Dpackage=org.openhab.binding.%BindingIdInLowerCase% -Dversion=%OpenhabVersion% -DbindingId=%BindingIdInLowerCase% -DbindingIdCamelCase=%BindingIdInCamelCase% -DvendorName=openHAB -Dnamespace=org.openhab -Dauthor="%Author%" -DgithubUser="%GithubUser%"
 
 COPY ..\src\etc\NOTICE org.openhab.binding.%BindingIdInLowerCase%\
 
+:: temporary fix
+:: TODO replace ${project.version} by ${ohc.version} in src/main/feature/feature.xml
+

```

Comment:
According to https://stackoverflow.com/questions/127318/is-there-any-sed-like-utility-for-cmd-exe this `@powershell --command "get-content  org.openhab.binding.%BindingIdInLowerCase%\src\main\feature\feature.xml | %{$_ -replace \"${project.version}\",\"$[ohc.version\"}"` should work